src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -34,12 +34,13 @@ public async Task<ReviewListItemModel> GetReviewAsync(string reviewId)
             return await _reviewsContainer.ReadItemAsync<ReviewListItemModel>(reviewId, new PartitionKey(reviewId));
         }
 
-        public async Task<ReviewListItemModel> GetReviewAsync(string language, string packageName, bool isClosed = false)
+        public async Task<ReviewListItemModel> GetReviewAsync(string language, string packageName, bool? isClosed = false)
```

Comment:
Will `isClosed` be null (unless explicitly passed) since default value is false?

Reply:
Default value is false. Since we usually want to fetch open reviews. One can pass null in the case where you want both open and closed Or true to get only closed reviews.